packages/neon/neon/lib/src/pages/home.dart

```diff
@@ -107,28 +107,22 @@ class _HomePageState extends State<HomePage> {
               const appBar = NeonAppBar();
 
               Widget body = Builder(
-                builder: (final context) => Column(
-                  children: [
-                    if (appImplementations.data != null) ...[
-                      if (appImplementations.data!.isEmpty) ...[
-                        Expanded(
-                          child: Center(
-                            child: Text(
-                              AppLocalizations.of(context).errorNoCompatibleNextcloudAppsFound,
-                              textAlign: TextAlign.center,
-                            ),
-                          ),
-                        ),
-                      ] else ...[
-                        if (activeAppIDSnapshot.hasData) ...[
-                          Expanded(
-                            child: appImplementations.data!.find(activeAppIDSnapshot.data!)!.page,
-                          ),
-                        ],
-                      ],
-                    ],
-                  ],
-                ),
+                builder: (final context) {
+                  if (!appImplementations.hasData || !activeAppIDSnapshot.hasData) {
```

Comment:
we shouldn't check for `!activeAppIDSnapshot.hasData` here.

When we don't have any supported app (appImplementations.data!.isEmpty) we also don't have an active app.
This check prevents us from showing the noapps view.